UPGRADE-1.7.md

```diff
@@ -50,7 +52,6 @@
    
 6. Copy migration files into `src/Migrations`:
 
-    - [Version20191119131635.php](https://raw.githubusercontent.com/Sylius/Sylius-Standard/1.7/src/Migrations/Version20191119131635.php)
```

Comment:
This will make the other migration fail while trying to revert the first one: https://github.com/Sylius/Sylius-Standard/blob/1.7/src/Migrations/Version20200301170604.php#L29

There are two possible solutions:
1. Generate one migration instead of those two
2. Leave these two migrations

If we go with 1., all projects on v1.7.0-ALPHA.2 will be much harder to upgrade.

Reply:
👍 I didn't notice this tiny `DROP type` 😄